Packs/CommonScripts/Scripts/ExtractAttackPattern/ExtractAttackPattern.py

```diff
@@ -0,0 +1,40 @@
+import demistomock as demisto
+from CommonServerPython import *
+
+import requests
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+
+def is_valid_attack_pattern(items):
+
+    try:
+        results = demisto.executeCommand('mitre-get-attack-pattern-value', {'attack_ids': items})
+        list_contents = results[0]['Contents']
+        values = [content.get('value') for content in list_contents]
+        return values if values else False
+
+    except ValueError as e:
+        if 'verify you have proper integration enabled to support it' in str(e):
+            demisto.info('Unsupported Command : mitre-get-attack-pattern-value, '
+                         'verify you have proper integration (MITRE ATTACK v2) enabled to support it. '
+                         'This Is needed in order to auto extract MITRE IDs and translate them to Attack Pattern IOCs')
+        return False
+    except Exception:
+        return False
```

Comment:
this will hide any other exception message. it should be logs as error with its message.

Reply:
done